### PR TITLE
Ignore positions recalc

### PR DESCRIPTION
**Pull request by dy**
This speeds up updating positions for https://github.com/plotly/plotly.js/pull/1657#issuecomment-307583657

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +28,8 @@
   "dependencies": {
     "binary-search-bounds": "^2.0.3",
     "clamp": "^1.0.1",
-    "color-id": "^1.0.0",
-    "font-atlas-sdf": "^1.2.0",
+    "color-id": "^1.1.0",
+    "font-atlas-sdf": "^1.3.1",
```
Comment (etpinard) [inline package.json]: Judging from 

https://github.com/dfcreative/font-atlas-sdf/commits/master

this should be `^1.3.2`, right?

**Comment by etpinard** [comment]
@dfcreative looking good!  All tests are passing off this branch. Moreover, I've played around with lasso/select off https://github.com/plotly/plotly.js/pull/1657 and the improvements are noticeable (maybe @monfera could confirm if he has the time :smile: )

Let's leave this PR open until https://github.com/plotly/plotly.js/pull/1657 is `status: reviewable` (I still need to add a few jasmine tests).

Thanks!

**Comment by etpinard** [comment]
All right, I couldn't find any broken behavior off this branch :tada: 

@dfcreative can you merge and publish this thing?

**Comment by dy** [comment]
@etpinard done!

**Comment by etpinard** [comment]
Beautiful. Thanks!

**Pull request closed without merging**